packages/@o3r/components/src/tools/placeholder/placeholder.component.ts

```diff
@@ -30,7 +31,11 @@ export class PlaceholderComponent implements OnInit, OnDestroy {
 
   private subscription = new Subscription();
 
-  private id$ = new ReplaySubject<string>(1);
+  private id$ = new BehaviorSubject<string | undefined>(undefined);
+
+  private readonly afterViewInit$ = new Subject<void>();
+
+  private readonly messages$ = new ReplaySubject<PlaceholderLoadingStatus>(1);
```

Comment:
Not sure it required to be `Replay` regarding the implementation of `sample` that already have a memory but no problem neither to tbe `Repley` so OK

Reply:
If not used, the unit tests are failing so I am not sure of that. I prefer leaving the replay subject in that case